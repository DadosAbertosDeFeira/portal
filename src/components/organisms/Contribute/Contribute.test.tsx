import { render } from "@testing-library/react";

import { Contribute } from ".";

describe("Contribute", () => {
  const makeSut = () => {
    return render(<Contribute />);
  };

  it("renders correctly", () => {
    const { asFragment } = makeSut();
    expect(asFragment).toMatchSnapshot();
  });
});
