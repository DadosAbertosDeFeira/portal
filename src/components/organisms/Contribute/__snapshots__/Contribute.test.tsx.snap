// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contribute renders correctly 1`] = `[Function]`;
